rotkehlchen/globaldb/upgrades/v2_v3.py

```diff
@@ -328,12 +328,21 @@ def translate_assets_in_price_table(cursor: 'DBCursor') -> List[Tuple[str, str,
     Translate the asset ids in the price table.
 
     Also drop all non manually input asset prices since otherwise this upgrade
-    will take forever. A heavily used globaldb
+    will take forever for a heavily used globaldb
     """
-    # cursor.execute('DELETE from price_history WHERE source_type!="A"')
+    assets = (
+        'ETH',
+        'ETH2',
```

Comment:
Maybe we should be storing the ETH2 price as ETH? Basically we are duplicating information here

Reply:
I think we maybe do so. But just putting it here just in case.